packages/gatsby/cache-dir/json-store.js

```diff
@@ -4,7 +4,10 @@ import PageRenderer from "./page-renderer"
 import { StaticQueryContext } from "gatsby"
 import socketIo, { getStaticQueryData, getPageQueryData } from "./socketIo"
 
-const getPathFromProps = props => props.pageResources?.page?.path
+const getPathFromProps = props =>
```

Comment:
Why revert this?

Reply:
Maybe I don't need to revert this, I thought this needed something like `transform-optional-chaining` plugin. I'll re-check it.